docs/types/reference-types.rst

```diff
@@ -550,3 +550,9 @@ members of the local variable actually write to the state.
 Of course, you can also directly access the members of the struct without
 assigning it to a local variable, as in
 ``campaigns[campaignID].amount = 0``.
+
+.. note::
+    In previous solidity versions, assigments which create memory-structs that contain a mapping
```

Comment:
A typo found by the[`chk_spelling` CI job](https://app.circleci.com/pipelines/github/ethereum/solidity/10828/workflows/0717982b-b997-44b7-9ece-ee1e37cbcaf3/jobs/524087):

```suggestion
    In previous solidity versions, assignments which create memory-structs that contain a mapping
```

Reply:
I was a bit surprised by this, my understanding was that when running `scripts/docs.sh` I would get notified about errors like this - I guess thiese checks are just part of the CI (?)

Reply:
Yeah, this is separate. `docs.sh` actually does not run any checks (except the built-in Sphinx).

To run it locally you'd need to use the Python `codespell` package with `scripts/codespell_whitelist.txt` like the [CI job does](https://github.com/ethereum/solidity/blob/v0.7.5/.circleci/config.yml#L288-L303).

Running it as a part of `docs.sh` sounds like a nice improvement. Docs already have Python dependencies listed in `requirements.txt` so adding another one would not be a problem. I think I'll add this when I have a spare moment. Or maybe you would like to try to submit a PR for that? :)